pyaedt/modeler/cad/Modeler.py

```diff
@@ -890,6 +897,37 @@ def quaternion(self):
             del self._modeler._app.variable_manager["temp_var"]
         return self._quaternion
 
+    @property
+    def origin(self):
+        """Returns the Coordinate System origin in model units.
```

Comment:
```suggestion
        """Coordinate system origin in model units.
```
Property descriptions don't start with verbs. According to Alex K., properties are "noun" descriptions.